blueocean-dashboard/src/main/js/creation/github/GithubFlowManager.js

```diff
@@ -129,65 +126,67 @@ export default class GithubFlowManager extends FlowManager {
     }
 
     onInitialized() {
-        this.findExistingCredential();
+        this._renderCredentialsStep();
         this.setPlaceholders('Complete');
     }
 
     destroy() {
         this._cleanupListeners();
     }
 
+    getScmId() {
+        return 'github';
+    }
+
     getApiUrl() {
         // backend will default to api.github.com
         return null;
     }
 
-    findExistingCredential() {
-        return this.accessTokenManager.findExistingCredential(this.getApiUrl())
-            .then(waitAtLeast(MIN_DELAY))
```

Comment:
Are these delays part of the GH api, or just slowing things down so the user can see what's going on?